src/test/resources/1332.xml

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE suite SYSTEM "http://beust.com/testng/testng-1.0.dtd" >
+  
+<suite name="A" verbose="1" >
+
+  <test name="Test1">
+    <classes>
+      <class name="test.reports.ReporterSample" />
```

Comment:
This java class is an[ existing class](https://github.com/cbeust/testng/blob/master/src/test/java/test/reports/ReporterSample.java). I am just re-using it.